docs/specs/restore.yml

```
@@ -204,10 +204,14 @@ repos:
 cache:
   .lock.json: |
     {
-      "git":{
-         "https://docs.com/lock1/c#master":{
-            "commit":"8720d784fb99abcbdfa922025b8bcc060f0f622a"
-         }
+      "git":
+      {
+        "https://docs.com/lock1/c#master":
```

use array over object, the object key contains duplicated info
```suggestion
        "git": [ { url, branch, commit }, { url, branch, commit } ]
```

---

will change that later because it requires backward compatibility